### ISSUE TITLE
Incorrect completion information with the no ncurses option 

### ISSUE THREAD

**Issue by smaret** [lime-rt/lime#54]
When LIME in started with the `-n` option, it says that the grid build is 99.9% completed, but then it takes a while to finish:

```
cd example
../lime -n model.c 
*** LIME, The versatile line modeling engine, Ver. 1.5
   Building grid: 99.90 percent done
```

Also the photon propagation completion cycles from 0% to 100% and back several times before completion.

**Comment by smaret** [comment]
I am continuing the discussion from #53 here. 
@imckstewart : what do you mean by "fixes in the latest no curses upload" ? I can find any fix in your repo.

**Comment by allegroLeiden** [comment]
Are you looking in allegroLeiden? If I look at messages.c in the nocurses branch I can see e.g. a call to fflush(stdout) at line 135.

**Comment by smaret** [comment]
Found it, thanks. Fixed with 218f5e72194818f603705a444f8afdf9724f184f.
